config/twill.php

```diff
@@ -156,6 +156,16 @@
      */
     'load_default_migrations' => true,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Twill Publish date formatting configuration
```

Comment:
After a more thorough review, I realized those configurations value were purposefully not set because the Vue components are defaulting to a computed format that takes the user locale into account.

Reply:
Removed it.